docs/reference/api/docker_remote_api.md

```diff
@@ -24,6 +24,26 @@ client must have `root` access to interact with the daemon. If a group named
 `docker` exists on your system, `docker` applies ownership of the socket to the
 group.
 
+To interact with the remote API using [cURL](http://curl.haxx.se/download.html) against the unix socket, you must have 
+a version of curl greater than 7.40, after which the --unix-socket flag was added. 
+
+For example, if the Docker daemon is listening on the default socket, then you can
+use curl against the remote API to get information on running containers by:
+```
+curl --unix-socket /var/run/docker.sock http://containers/json
+```
+
+It is also possible to bind the Docker daemon to a TCP port at startup by:
```

Comment:
@thaJeztah I agree with you.  The ins and outs of using a different sock should be addressed elsewhere.  I'd change this to:

```markdown
To interact with the remote API using [cURL](http://curl.haxx.se/download.html).
You run `curl` against the socket where the Docker daemon is listening. For this
reason, must use curl 7.40 (or greater) as these versions have the
`--unix-socket` flag is available. To run `curl` against the deamon on
the default socket, use the following:
```
```
curl --unix-socket /var/run/docker.sock http://containers/json
```
```markdown
If you bound the Docker daemon to a different, you would reference that alternative port in your curl rather than the default.
```